src/main/java/frc/team2412/robot/commands/drivebase/DriveCommand.java

```diff
@@ -75,30 +79,34 @@ public void execute() {
 								- (speedLimiter.getAsDouble()
 										* (1 - triggerModifierEntry.getDouble(TRIGGER_MODIFIER_DEFAULT))));
 
-		double x = deadbandCorrection(-forward.getAsDouble());
-		double y = deadbandCorrection(strafe.getAsDouble());
-		double rot = deadbandCorrection(-rotation.getAsDouble());
+		if (autoBalance.getAsBoolean()) {
+			drivebaseSubsystem.drive(0, 0, new Rotation2d(), fieldOrientedEntry.getBoolean(true), true);
```

Comment:
Once we have vision updating pose and we have a good understanding of our location, we should put an item on the to do list (future PR) to make sure we don't drive off the platform and lose points when autobalancing.
Another item for the to do list, if we are checking pose, it might be nice to align the rotation to be most conducive to autobalancing.

Reply:
If we try to align the robot, it might become cleaner to have a separate command that temporarily interrupts the default `DriveCommand` so we can express the sequence through command groups instead of within `execute`.
```java
driveController.leftBumper().whileTrue(align.andThen(balance));
```